AppInsightsWithWebJob/ContinuousJob.cs

```diff
@@ -1,39 +1,45 @@
-﻿using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-
-namespace AppInsightsWithWebJob
+﻿namespace AppInsightsWithWebJob
 {
-    public class ContinuousJob : IHostedService
+    using System;
```

Comment:
Sorry, move them out of the namespace not into it. At least thats the default style.

Reply:
Oh, ignore that. Old habit of what we do internally.